fastlane/docs/FAQs.md

```diff
@@ -21,6 +21,24 @@ If you use `rvm`, try the following
 rvm osx-ssl-certs update all
 ```
 
+### Fastlane is slow (to start)
+
+Fastlane uses rubygems and [rubygems is known to be slow itself](http://www.sitepoint.com/rubygems-slow/)
+
+If you see abnormal slowdown when you run fastlane, here are 2 known ways to reduce this problem:
+
+* if you use fatlane with a Gemfile, use `bundle exec` to reduce the amount of searched gems. E.g.
+
+```
+bundle exec fastlane actions
```

Comment:
We can also link to https://guides.cocoapods.org/using/a-gemfile.html for more information